### PR TITLE
Random stuff

### PR DESCRIPTION
**Pull request by fewizz**

### CODE REVIEW

**Comment by asiekierka** [comment]
Since you're new here, I need to ask you the obligatory question: Is the contribution your own work and not based on looking up existing mappings?

**Comment by fewizz** [comment]
Only my, and not based.

**Comment by Darkhax** [comment]
The looks alright. Besides the merge conflicts, LGTM? :new_moon_with_face:

File: mappings/net/minecraft/world/chunk/ChunkBlockInfo.mapping [hunk 1]
```diff
@@ -1,4 +1,11 @@
 CLASS none/atn net/minecraft/world/chunk/ChunkBlockInfo
+	FIELD a height I
+	FIELD b blockCount I
+	FIELD c tickableBlockCount I
+	FIELD e metaDataArray Lnone/ati;
```
Comment (asiekierka) [inline mappings/net/minecraft/world/chunk/ChunkBlockInfo.mapping]: metadataArray?

Comment (fewizz) [reply mappings/net/minecraft/world/chunk/ChunkBlockInfo.mapping]: I will rename, fix conflicts, and add more names. Dont't merge atm. Thanks.

**Comment by fewizz** [comment]
I merged changes, and fixed conflict.

**Pull request merged**